doc/connectivity/networking/api/wifi.rst

```diff
@@ -0,0 +1,23 @@
+.. _wifi_mgmt:
+
+Wi-Fi Management
+################
+
+Overview
+========
+
+The Wi-Fi management API is used to manage Wi-Fi networks. It supports below modes:
+
+* IEEE802.11 Station (STA)
+* IEEE802.11 Access Point (AP)
+
+The Wi-Fi management API is implemented in the `wifi_mgmt` module as a networking L2 stack.
```

Comment:
> implemented in the `wifi_mgmt` module as a networking L2 stack

This is a bit unclear to me, can you elaborate on what did you mean here exactly? What does it mean that "wifi_mgmt" is implemented as a "networking L2 stack"?

Reply:
By `stack` I was referring to an entity part of kernel and sitting on top of L2 which for Wi-Fi could be Etherne/Offloaded